src/Utilities/StringHelper.cs

```diff
@@ -0,0 +1,26 @@
+//-----------------------------------------------------------------------------
+// <copyright file="StringHelper.cs" company="WheelMUD Development Team">
+//   Copyright (c) WheelMUD Development Team.  See LICENSE.txt.  This file is 
+//   subject to the Microsoft Public License.  All other rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace WheelMUD.Utilities
+{
+    public static class StringHelper
+    {
+        /// <summary>
+        /// Returns a string with ',' or 'and' depending on item count.
+        /// </summary>
```

Comment:
nit:  `/// <remarks>Output example: "apples, oranges and bananas".</remarks>` ?
nit: Prefer to either fill the other parts or delete them too.
nit: Would be a good candidate for some small quick unit tests to cover some cases (no items, 1, 2, 3, 4), as it's just hard enough to know if it's right from looking at it.